src/lib/ipr.js

```diff
@@ -42,6 +46,79 @@ function createPacketAndCondition (params) {
 }
 
 /**
+  * Create an encoded IPR for use in the IPR transport protocol
+  *
+  * @param {Object} params Parameters for encoding IPR
+  * @param {String} params.packet ILP packet of this IPR
+  * @param {String} params.condition condition of this IPR
+  *
+  * @return {Buffer} encoded IPR buffer
+  */
+function encodeIPR ({
+  packet,
+  condition
+}) {
+  assert(packet, 'params.packet must be specified')
+  assert(condition, 'params.condition must be specified')
+
+  const packetBuf = Buffer.from(packet, 'base64')
+  const conditionBuf = Buffer.from(condition, 'base64')
+
+  assert(conditionBuf.length === 32, 'params.condition must encode 32 bytes')
+
+  const writer = new oer.Writer()
+
+  writer.writeUInt8(IPR_VERSION)
```

Comment:
@justmoon the ILP packets have a type and then the whole thing is in an OER octet string (so it has the length prefix before it) -- should IPR have either of those things?